lib/appsignal/logger.rb

```diff
@@ -23,6 +23,7 @@ class Logger < ::Logger
     # @param level Log level to filter with
     # @return [void]
     def initialize(group, level: INFO, format: PLAINTEXT)
+      super(group, level: level)
```

Comment:
```suggestion
      super(group, level:)
```

Reply:
I used the "old syntax" in case they need to support older Ruby versions. I'm not sure how far back they want to support.

Reply:
Ah, makes sense. You'll need to use hash rocket syntax then to comply with RuboCop.